proteobench/utils/plotting/plot.py

```diff
@@ -66,7 +66,6 @@ def plot_fold_change_histogram(result_df: pd.DataFrame, species_ratio: dict) ->
     @staticmethod
     def plot_metric(
         benchmark_metrics_df: pd.DataFrame,
-        reinitialize_table: bool = False,
         software_colors: dict = {
```

Comment:
not for now, but I think that we should consider having the software colors defined somewhere else. Maybe in the module .toml?